components/elective/ElectiveStudentListCard.tsx

```diff
@@ -114,24 +79,11 @@ const ElectiveStudentListCard: StylableFC<{
           transition={transition(DURATION.medium2, EASING.standardDecelerate)}
           className="space-y-1 pb-4 pr-4"
         >
-          {students.map((student) => (
-            <PersonCard
-              key={student.id}
-              person={student}
-              options={{
-                // Show an asterisk if the Student is enrolled via
-                // randomization.
-                suffix: electiveSubject.randomized_students.some(
-                  (randomizedStudent) => student.id === randomizedStudent.id,
-                ) ? (
-                  <span title={t("randomized")} className="text-tertiary">
-                    *
-                  </span>
-                ) : undefined,
-              }}
-              className="w-full !border-0 !bg-surface-container"
-            />
-          ))}
+          <ElectiveStudentList
+            electiveSubject={electiveSubject}
+            query={query}
+            className="[&_button]:!border-0 [&_button]:!bg-surface-container"
+          />
```

Comment:
Turning the code into a component is good for making the code more flexible for future use.